dirsrvtests/tests/suites/paged_results/paged_results_test.py

```diff
@@ -1117,7 +1123,7 @@ def test_multi_suffix_search(topology_st, create_user, new_suffixes):
     try:
         req_ctrl = SimplePagedResultsControl(True, size=page_size, cookie='')
 
-        all_results = paged_search(topology_st.standalone, NEW_SUFFIX_1, [req_ctrl], search_flt, searchreq_attrlist)
+        all_results = paged_search(topology_st.standalone, NEW_SUFFIX_1, [req_ctrl], search_flt, searchreq_attrlist, 0, True)
```

Comment:
Should better name explicitly the optional attribute instead on relying on its position:
`all_results = paged_search(topology_st.standalone, NEW_SUFFIX_1, [req_ctrl], search_flt, searchreq_attrlist, avoid_race_con=True)`